ideahacks/helpers/userFunctions.js

```diff
@@ -0,0 +1,31 @@
+const formatUser = user => {
+  if (user.hasTeam) {
+    user.hasATeam = true
+  }
+  user.hasNoTeam = true
+  if (user.vehicleNeed) {
+    user.doesNeedVehicle = true
+  }
+  user.doesNotNeedVehicle = true
+
+  let emailSize = user.teammates.length
```

Comment:
Can you change `emailSize` ==> `numberOfTeammates`. emailSize is an unintuitive variable name.